pyrefinebio/sample.py

```diff
@@ -137,43 +137,62 @@ def search(cls, **kwargs):
         returns: list of samples.
 
         Parameters:
-            ordering (str):                  which field to use when ordering the results
-            title (str):
-            organism (str):
-            source_database (str):
-            source_archive_url (str):
-            has_raw (str):
-            platform_name (str):
-            technology (str):
-            manufacturer (str):
-            sex (str):
-            age (number):
-            specimen_part (str):
-            genotype (str):
-            disease (str):
-            disease_stage (str):
+            ordering (str): which field to use when ordering the results
+
+            title (str): filter based on the sample's title
+
+            organism (str): filter based on the organism that the sample was taken from
+
+            source_database (str): filter based on the database that the sample was taken from
+
+            source_archive_url (str): filter based on sample's source url
+
+            has_raw (bool): filter based on if the sample
+
+            platform_name (str): filter based on the name of the platform that was used to create the sample
+
+            technology (str): filter based on the technology that was used to make the sample
+
+            manufacturer (str): filter based on the manufacturer of the technology that was used to create the sample
+
+            sex (str): filter based on the sex of the organism that the sample was taken from
+
+            age (number): filter based on the age of the organism that the sample was taken from
+
+            specimen_part (str): filter based on the part of the specimen that the sample was taken from
+
+            genotype (str): filter based on the sample subject's genotype 
+
+            disease (str): filter based on the sample subject's disease
+
+            disease_stage (str): filter based on the stage of the sample subject's disease
+
             cell_line (str):
-            treatment (str):
-            race (str):
-            subject (str):
+
+            treatment (str): filter based on the treatment used for the sample subject's disease
+
+            race (str): filter based on the race of the sample's subject 
+
+            subject (str): filter based on the name of the sample's subject
+
             compound (str):
```

Comment:
wasn't sure what to put for `cell_line` and `compound`

Reply:
I think for compound we can say "filter based on the compound the sample was treated with"

I can't say for sure about the cell line though. The Cell Line for https://www.refine.bio/experiments/SRP003820/gse24545-canton-s-adult-ovaries is `canton s`. @jaclyn-taroni would it be accurate to say "the name of the cell line" or "the type of the cell line"?

Reply:
It depends on the study, which I think means we want to be more general like `filter on the basis of the cell line field`. I think the docstrings in this file probably could use a scientific review.

Reply:
👍  yeah I definitely think that probably all of the docstrings should go through a scientific review.

Reply:
👍  yeah, I think that probably all of the docstrings should go though a scientific review tbh

Reply:
👍  sounds good - scientific review of this pull request is on my list this week!